playbooks/roles/openvpn/tasks/main.yml

```diff
@@ -126,13 +126,37 @@
 - name: Copy OpenVPN configuration file into place
   template:
     src: etc_openvpn_server.conf.j2
-    dest: /etc/openvpn/server.conf
+    dest: "{{ openvpn_path }}/server.conf"
 
 - name: Copy OpenVPN UDP configuration file into place
   template:
     src: etc_openvpn_server_udp.conf.j2
-    dest: /etc/openvpn/server-udp.conf
-  notify: Restart OpenVPN
+    dest: "{{ openvpn_path }}/server-udp.conf"
+
+- name: Stop and disable openvpn.service
```

Comment:
It might avoid some confusion if this task name specified that it was stopping and disabling the stock/upstream service.